### DISCUSSION TITLE
Advices:  some description about Middleware orders in the docs.

### DISCUSSION THREAD

**Discussion by me-v2** [fastapi/fastapi#8761] [Questions] [answered]
in the docs, cann't find some detail about  order of middleware .

**Comment by mawassk** [comment]
They will be executed in the order you add them. See implementation: https://github.com/encode/starlette/blob/master/starlette/applications.py

**Comment by skfarhat** [comment]
I think this detail is worth adding in [the documentation](https://fastapi.tiangolo.com/tutorial/middleware/). Indeed I thought it was the case that middleware is applied in order but couldn't find a reference, then came hunting in Github Issues :)

**Comment by YuriiMotov** [comment, marked as answer]
Recently added section in docs: https://fastapi.tiangolo.com/tutorial/middleware/#multiple-middleware-execution-order
